### PR TITLE
[zh-cn] sync kubeadm-upgrade.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -110,12 +110,38 @@ The upgrade workflow at high level is the following:
...
+etcd upgrade, in-flight requests to the server will stall while the new etcd
+static pod is restarting. As a workaround, it is possible to actively stop the
+`kube-apiserver` process a few seconds before starting the `kubeadm upgrade
+apply` command. This permits to complete in-flight requests and close existing
+connections, and minimizes the consequence of the etcd downtime. This can be
+done as follows on control plane nodes:
+-->
+### 升级 etcd 时的注意事项
+
+由于 `kube-apiserver` 静态 Pod 始终在运行（即使节点资源耗尽），
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:127]:
```suggestion
由于 `kube-apiserver` 静态 Pod 始终在运行（即使你已经执行了腾空节点的操作），
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 2]
```diff
@@ -110,12 +110,38 @@ The upgrade workflow at high level is the following:
...
+因此当你执行包括 etcd 升级在内的 kubeadm 升级时，对服务器的正在进行的请求将停止，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:128]:
```suggestion
因此当你执行包括 etcd 升级在内的 kubeadm 升级时，对服务器的正在进行的请求将停滞，
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 3]
```diff
@@ -110,12 +110,38 @@ The upgrade workflow at high level is the following:
...
+而新的 etcd 静态 Pod 正在重新启动。作为解决方法，可以在运行 `kubeadm upgrade apply`
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:129]:
```suggestion
因为要重新启动新的 etcd 静态 Pod。作为一种解决方法，可以在运行 `kubeadm upgrade apply`
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 4]
```diff
@@ -110,12 +110,38 @@ The upgrade workflow at high level is the following:
...
+命令之前主动停止 `kube-apiserver` 进程几秒钟。这允许完成正在进行的请求并关闭现有连接，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:130]:
```suggestion
命令之前主动停止 `kube-apiserver` 进程几秒钟。这样可以允许正在进行的请求完成处理并关闭现有连接，
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 5]
```diff
@@ -110,12 +110,38 @@ The upgrade workflow at high level is the following:
...
+并最大限度地减少 etcd 停机的后果。这可以在控制平面节点上按如下方式完成：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:131]:
```suggestion
并最大限度地减少 etcd 停机的后果。此操作可以在控制平面节点上按如下方式完成：
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 6]
```diff
@@ -110,12 +110,38 @@ The upgrade workflow at high level is the following:
...
+
+```shell
+killall -s SIGTERM kube-apiserver # trigger a graceful kube-apiserver shutdown
+sleep 20 # wait a little bit to permit completing in-flight requests
+kubeadm upgrade ... # execute a kubeadm upgrade command
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:136]: 注释也翻译一下

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 7]
```diff
@@ -110,12 +110,44 @@ The upgrade workflow at high level is the following:
...
+static pod is restarting. As a workaround, it is possible to actively stop the
+`kube-apiserver` process a few seconds before starting the `kubeadm upgrade
+apply` command. This permits to complete in-flight requests and close existing
+connections, and minimizes the consequence of the etcd downtime. This can be
+done as follows on control plane nodes:
+-->
+### 升级 etcd 时的注意事项
+
+由于 `kube-apiserver` 静态 Pod 始终在运行（即使你已经执行了腾空节点的操作），
+因此当你执行包括 etcd 升级在内的 kubeadm 升级时，对服务器的正在进行的请求将停滞，
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:128]:
```suggestion
因此当你执行包括 etcd 升级在内的 kubeadm 升级时，对服务器正在进行的请求将停滞，
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 8]
```diff
@@ -110,12 +110,44 @@ The upgrade workflow at high level is the following:
...
+并最大限度地减少 etcd 停机的后果。此操作可以在控制平面节点上按如下方式完成：
+
+<!--
+```shell
+# trigger a graceful kube-apiserver shutdown
+# wait a little bit to permit completing in-flight requests
+# execute a kubeadm upgrade command
+-->
+```shell
+killall -s SIGTERM kube-apiserver # 触发 kube-apiserver 正常关闭
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:140]:
```suggestion
killall -s SIGTERM kube-apiserver # 触发 kube-apiserver 体面关闭
```

**Review by windsonsea — COMMENTED** [review]
nit

**Pull request merged**